fluent-plugin-google-cloud.gemspec

```diff
@@ -29,7 +29,9 @@ eos
   gem.add_runtime_dependency 'json', '2.2.0'
 
   gem.add_development_dependency 'mocha', '1.9.0'
-  gem.add_development_dependency 'prometheus-client', '0.10.0'
+  # Keep this the same as in
+  # https://github.com/fluent/fluent-plugin-prometheus/blob/master/fluent-plugin-prometheus.gemspec
```

Comment:
Can we add a warning here to mention the issue with `prometheus-client` 0.10 so that we don't upgrade by accident? It would be even better if it mentions a ticket / bug to monitor the root cause fix so we know when it's safe to upgrade.

Reply:
Re. comment, I think the goal here clear: keep the import the same, to avoid runtime conflicts. If anyone is feeling curious about how we learned that, they could use git blame to find this PR.

If you feel strongly about that, please suggest the wording you'd like to see and I'll apply the change, or make the change directly to the branch. This is currently breaking our integration test and I was hoping to submit yesterday so we could make a release on time next week.

We are working around the fact that either Ruby doesn't enforce compatible runtime dependencies or the FluentD plugin mechanism accidentally breaks that by loading the plugins at runtime. Upgrading to 0.10.0 would be totally fine, if the other plug-in upgrades to it as well. If at some point we must upgrade for any reason, we could make the change to the prometheus plugin as well, it's just a couple lines.

As for the bug, can you talk more about the benefits of filing a bug to track this? I'm concerned that we would have to keep two things up to date: the bug and this comment.

Reply:
Oh, i completely misunderstood why we reverted to `0.9.X`. I thought it was the `0.10.0` gem itself that included some bug and broke us, so I was asking for _that_ bug ticket. If it's just mismatching versions, that's totally fine. I did not realize it was a gem version conflict thing. It all makes sense now. Thanks for the clarification.